components/ui/forms/date-picker/date-picker-input.tsx

```
@@ -0,0 +1,540 @@
+                        selectionStart,
+                        selectionEnd,
+                        "none"
+                    );
+                });
+            } else {
+                element.setSelectionRange(selectionStart, selectionEnd, "none");
+            }
+        }
+    });
+}
+export function useEventCallback<T extends Function>(fn: T, deps: any[]) {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)